src/twisted/names/authority.py

```diff
@@ -14,14 +14,24 @@
 from twisted.names import dns, error, common
 from twisted.internet import defer
 from twisted.python import failure
-from twisted.python.compat import execfile
+from twisted.python.compat import execfile, nativeString, _PY3
+from twisted.python.filepath import FilePath
 
 
-def getSerial(filename = '/tmp/twisted-names.serial'):
-    """Return a monotonically increasing (across program runs) integer.
+
+def getSerial(filename='/tmp/twisted-names.serial'):
```

Comment:
since this is targeting python3, is filename a native string, bytes or unicode?

with the move to python3 I think that it would help to be explicit if something is text or data.